components/project/list-sc.js

```diff
@@ -9,32 +9,20 @@ const ProjectListSC = styled.div`
   grid-gap: 20px;
 
   ${media.phone`
-    display: grid;
```

Comment:
@Magitron I noticed some CSS rules were used across all screen sizes. When I tried to remove them and see if they can be applied across all sizes, it just worked 😄

Reply:
Ah yes! Thank you!